src/App.js

```diff
@@ -1,28 +1,43 @@
-import React, { useEffect, useState } from "react";
 import "./App.css";
 import CurrencyRow from "./components/CurrencyRow";
 import CurrencyConverterText from "./components/CurrencyConverterText";
 import TimeStamp from "./components/TimeStamp";
-import useCurrencyConverter from "./components/useCurrencyConverter";
+import useCurrencyConverter from "./hooks/useCurrencyConverter";
+import useFetch from "./hooks/useFetch";
+import React, { useState } from "react";
 
-const endpoint = "latest";
-const access_key = process.env.REACT_APP_EXCHANGE_RATES_API_KEY;
+const params = {
+  access_key: process.env.REACT_APP_EXCHANGE_RATES_API_KEY,
+  endpoint: "latest",
+};
+
+const BASE_URL =
+  "http://api.exchangeratesapi.io/v1/" +
+  params.endpoint +
+  "?access_key=" +
+  params.access_key;
 
 function App() {
+  const [amount, setAmount] = useState(1);
```

Comment:
what is the purpose of this amount variable here and why is it set to 1 in the first place

Reply:
The purpose of this variable is the amount displayed in the upper input field, which is given the parameter of 1 in order to display a value of 1 as the initial number